pyproject.toml

```diff
@@ -0,0 +1,199 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: Free for non-commercial use',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+description = 'Benchmark tabular synthetic data generators using a variety of datasets'
+keywords = ['machine learning', 'synthetic data generation', 'benchmark', 'generative models']
+name = 'sdgym'
+version = '0.7.1.dev0'
+license = { text = 'BSL-1.1' }
+requires-python = '>=3.8,<3.12'
+readme = 'README.md'
+urls = { 'Homepage' = 'https://github.com/sdv-dev/SDGym' }
```

Comment:
Remove from here and put below
```suggestion
```

Reply:
Yes, done in [7fac59d](https://github.com/sdv-dev/SDGym/pull/269/commits/7fac59d7ad872f32dc3b7307944cfe1980724f17)